packages/version-bump/src/tests/utils.ts

```diff
@@ -31,7 +31,8 @@ export const getMockedFileContents = (path: string, version?: string): string =>
 type MainTestCase = {
 	type: BumpType;
 	inputVer: string;
-	outputVer?: string;
+	outputVer: string;
+	value?: string;
```

Comment:
maybe we can pick from `Input` type what is being used here, this way it's easier to deduct where else these types are being used; 
this is very optional suggestion, it's up to you

Reply:
As mentioned in [this comment](https://github.com/eventespresso/barista/issues/271#issuecomment-728211584), we want to be explicit about everything and not to deduce anything ourselves. The consumer will have total control on the inputs.